maven.html

```diff
@@ -19,14 +19,28 @@ <h2>Run HelloWorld using Maven</h2>
     &lt;plugin&gt;
         &lt;groupId&gt;org.openjfx&lt;/groupId&gt;
         &lt;artifactId&gt;javafx-maven-plugin&lt;/artifactId&gt;
-        &lt;version&gt;<span class="JFX_MVN_PLUGIN_VERSION">0.0.1</span>&lt;/version&gt;
+        &lt;version&gt;<span class="JFX_MVN_PLUGIN_VERSION">0.0.3</span>&lt;/version&gt;
```

Comment:
This is not necessary, there is a variable named `JFX_MVN_PLUGIN_VERSION` set in `gluon.js` with this value, precisely to avoid updating manually all occurrences whenever the versions change.